src/Raven.Studio/typescript/viewmodels/database/timeSeries/filterTimeSeries.ts

```diff
@@ -0,0 +1,104 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import dialog = require("plugins/dialog");
+import datePickerBindingHandler = require("common/bindingHelpers/datePickerBindingHandler");
+import generalUtils = require("common/generalUtils");
+
+class filterTimeSeries extends dialogViewModelBase {
+
+    setStartDate = ko.observable<boolean>(false);
+    setEndDate = ko.observable<boolean>(false);
+    
+    startDateToUse = ko.observable<number>(null);
+    endDateToUse = ko.observable<number>(null);
+    
+    startDateLocal = ko.observable<moment.Moment>();
+    endDateLocal = ko.observable<moment.Moment>();
```

Comment:
looks like we don't need to duplicate those variables

Reply:
done